@xen-orchestra/lite/src/locales/en.json

```diff
@@ -17,6 +17,7 @@
   "coming-soon": "Coming soon!",
   "community": "Community",
   "community-name": "{name} community",
+  "console": "Console",
```

Comment:
If the translation is the same, it can be used directly.

Reply:
I'm not quite certain what you're referring to.

Are you discussing the direct use of terms like `<div>Console</div>` without any translation?

If that's the case, I don't agree, as we might need to include more translations in the future that would vary.

Moreover, we have consistently translated words of this nature from the outset (e.g., Description, Documentation, Pause, etc.).